vault/dendron.topic.frontmatter.md

```diff
@@ -50,6 +50,8 @@ Note that not all the keys are written out to the Markdown file. Dendron merges
 
 A globally unique identifier for the note. This identifier is used for two things: (1) generating unique URLs when notes are published and (2) ensuring each note in the index is unique.
 
+A dendron id is lower case alphanumeric and 32 characters long. 
```

Comment:
Should we amend this a little bit?  Because a dendron id can still be upper case (particularly in existing notes prior to the code change).  Maybe insert the word 'default', i.e. "By default, a Dendron ID is a lower case..."